### ISSUE TITLE
vllm deploy the same query always same response. I cannot find where to config random seed

### ISSUE THREAD

**Issue by rocke2020** [QwenLM/Qwen#847]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

vllm 0.2.2 with pytorch 2.1, cuda 12.1, qwen/Qwen-7B-Chat
I successfully run the from 
`from examples.vllm_wrapper import vLLMWrapper` 
the same query always same response. 
I cannot find where to config random seed
I also fastly check vllm, there is no mention this random seed issue.
Do you encounter similar issue?
thanks!

My query(prompt) are 
   response, history = model.chat(query="你好",
                                   history=None)
    print('1,', response, flush=True)
    response, history = model.chat(query="给我讲一个年轻人奋斗创业最终取得成功的故事。",
                                   history=history)
    print('2,', response, flush=True)
    response, history = model.chat(query="给这个故事起一个标题",
                                   history=history)
    print('3,', response, flush=True)
    response, history = model.chat(query="再讲一个不同的创业故事,情节复杂和具体的长一点的故事，逆袭成功",
                                   history=history)
    print('4,', response, flush=True)

### 期望行为 | Expected Behavior

there are different response for the same prompt.

### 运行环境 | Environment

```Markdown
- OS: ubuntu 20.04
- Python: 3.10.13
- Transformers: 4.36.1
- PyTorch: 2.1
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`): 12.1
```

**Comment by hzhwcmhf** [comment]
Currently, vllm_wrapper initializes vLLM with seed=0. You can add the ``seed`` argument on this line:

https://github.com/QwenLM/Qwen/blob/0a5b3b6b9aec3a3e7b1c2bd93580f35cfe6546fb/examples/vllm_wrapper.py#L135-L141

**Comment by rocke2020** [comment]
thanks a lot.
